cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -2893,6 +2850,11 @@ pub(crate) mod tests {
             TEST_RESOLUTION_RESPONSE_GENERAL.clone(),
             TEST_RESOLUTION_RESPONSE_SYSTEMS_NOT_ON_SAME_PAGE.clone(),
         ] {
+            // It doesn't make sense to generate mocks since:
+            // - We're testing for future messages with unknown message
+            // - General isn't currently used serverside
+            // We could probably generate one for SYSTEMS_NOT_ON_SAME_PAGE if we
+            // wanted.
```

Comment:
Per sync discussion we should try to handcraft some yaml.  I'll come back to this and the other commented cases in followup